toolchain/lower/lower.cpp

```diff
@@ -12,18 +12,23 @@
 
 namespace Carbon::Lower {
 
+// TODO: Remove module_name argument.
```

Comment:
Looks like you already did this TODO :)

Reply:
Thanks, removed (I'd been waffling about whether to do it in this PR)